src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,62 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Prors = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: React.Dispatch<React.SetStateAction<{
+    id: string;
+    title: string;
+    content: string;
+  }>>,
+};
+
+export const Tabs: React.FC<Prors> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handlerTab = (tab: Tab) => {
+    if (tab !== selectedTab) {
```

Comment:
very dangerous comparison, try to avoid object comparisons. Use better ids here tab.id !== selectedTab.id